agents/craftassist/craftassist_agent.py

```diff
@@ -170,12 +173,15 @@ def init_inventory(self):
 
     def init_memory(self):
         """Intialize the agent memory and logging."""
+        low_level_data = self.low_level_data.copy()
+        low_level_data.update({'check_inside': heuristic_perception.check_inside})
```

Comment:
kind of nit, but why not just low_level_data["check_inside"] = heuristic_perception.check_inside

Reply:
Yes yes definitely, updating that :)